docs/REST API/rest-api.md

```diff
@@ -3630,6 +3642,80 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Lock host
+
+_Available in Fleet Premium_
+
+Sends a command to lock the specified macOS, Linux, or Windows host. The host is locked once it comes online.
+
+`POST /api/v1/fleet/hosts/:id/lock`
+
+#### Parameters
+
+| Name       | Type              | In   | Description                                                                   |
+| ---------- | ----------------- | ---- | ----------------------------------------------------------------------------- |
+| id | integer | path | **Required**. ID of the host to be locked. |
+
+#### Example
+
+`POST /api/v1/fleet/hosts/123/lock`
+
+##### Default response
+
+`Status: 200`
+
+### Unlock host
+
+_Available in Fleet Premium_
+
+Sends a command to unlock the specified Windows or Linux host, or retrieves the unlock PIN for a macOS host.
+
+`POST /api/v1/fleet/hosts/:id/unlock`
+
+#### Parameters
+
+| Name       | Type              | In   | Description                                                                   |
+| ---------- | ----------------- | ---- | ----------------------------------------------------------------------------- |
+| id | integer | path | **Required**. ID of the host to be unlocked. |
+
+#### Example
+
+`POST /api/v1/fleet/hosts/:id/unlock`
+
+##### Default response
+
+`Status: 200`
+
+Requests to unlock macOS hosts will also include the following response data:
+
+```json
+{
+  "host_id": 8,
+  "unlock_pin": 123456
+}
+```
+
+
+### Wipe host
+
+Sends a command to wipe the specified macOS, Windows, or Linux host. The host is wiped once it comes online.
+
+`POST /api/v1/fleet/hosts/:id/wipe`
```

Comment:
@noahtalerman @rachaelshaw same here, this endpoint supercedes/deprecates the old `POST /api/v1/fleet/mdm/hosts/:id/wipe`? It will remain supported but only for macOS?